api/yelp_beans/matching/match_utils.py

```diff
@@ -86,3 +88,96 @@ def get_previous_meetings(subscription, cooldown=None):
     disallowed_meetings = {tuple([meeting.id for meeting in meeting]) for meeting in disallowed_meetings}
 
     return disallowed_meetings
+
+
+def jaccard(list1, list2):
+    intersection = len(list(set(list1).intersection(list2)))
+    if intersection == 0:
+        return 1
+    else:
+        union = (len(list1) + len(list2)) - intersection
+        return float(intersection) / union
+
+
+def get_pairwise_distance(
```

Comment:
Would it be possible to make the attributes used configurable? I think it'd be great to have the choice of attributes to apply be something that can be configured differently for different subscriptions